spec/fabricators/order_fabricator.rb

```diff
@@ -4,6 +4,7 @@
     order.required_date = order.order_date + rand(1.month).seconds
     order.shipped_date = order.order_date + rand(1.month).seconds
   end
+  freight { Random.rand(0.0...1.0) }
```

Comment:
Freight to koszt dostawy - nie powininen być z przedziału 0-1. Może 0-100?

Reply:
A masz rację pomyliło mi się z discount :smile: